iterations_runners/iterations_runner.som

```diff
@@ -14,13 +14,10 @@ BenchmarkHarness = (
     )
 
     usage = (
-      'Iterations runner for SOM VMs.' println.
-      '' println.
-      'Executes a benchmark many times within a single process.' println.
-      '' println.
+      'Iterations runner for SOM VMs. \n' println.
```

Comment:
Are the trailing spaces at the end of lines intentional (after the `.` and before the `\n`)?

Reply:
No, they're the result of me not thinking. Try this: e4fbf31a45a8fcc68c4d35fe63ade2541e0062ea